f2/src/locales/en.json

```diff
@@ -281,20 +273,17 @@
   "suspectClues.title": "Suspect clues",
   "suspectClues.whatComesNext": "Next, you will be asked about supporting evidence.",
   "thankYouPage.createNewReport": "Create a new report",
-  "thankYouPage.feedback": "Have feedback?",
-  "thankYouPage.helpResource": "Where you can get help",
-  "thankYouPage.helpResource1": "Talk to someone and get support.",
-  "thankYouPage.helpResource2": "Find tips to protect yourself in the future.",
-  "thankYouPage.helpResource3": "Stay up-to-date on what scams and crimes are most common.",
-  "thankYouPage.otherReportOption": "You have other reporting options",
-  "thankYouPage.otherReportOption1": "Report directly to the place where the incident happened.",
-  "thankYouPage.otherReportOption2": "Report to your local police service, if you feel you are in immediate danger.",
-  "thankYouPage.otherReportOption3": "Report to your financial institution, if you lost money.",
-  "thankYouPage.otherReportOption4": "Report to both credit bureaus, <0>Equifax</0> and <1>TransUnion</1>, if your identity was stolen.",
+  "thankYouPage.feedback": "Have feedback to help us improve this site?",
+  "thankYouPage.helpResource": "Where you can get help:",
+  "thankYouPage.helpResource1": "Learn about how you can protect yourself in the future.",
+  "thankYouPage.helpResource2": "Stay up-to-date on what scams are most common.",
+  "thankYouPage.helpResource3": "Find tips on how to prevent cybercrime.",
   "thankYouPage.referenceNumber": "Your reference number is <0>{reference}</0>",
   "thankYouPage.safelyCloseWindow": "You can safely close this window",
   "thankYouPage.summary": "We've sent you an email with a summary of your report.",
   "thankYouPage.title": "Thank you for reporting",
+  "thankYouPage.whatNextHeading": "What happens next?",
+  "thankYouPage.whatNextParagraph": "We’ve sent you a confirmation email with this number. You can add more detail later by calling the Canadian Anti-Fraud Centre at 1-888-495-8501.",
```

Comment:
We have to make saying "We've sent you an email" conditional on whether they've provided their email address. Right now this is being done in the black "Thank you for reporting" box.

Reply:
💯 
Will correct 😄